helpers/pginit.js

```diff
@@ -0,0 +1,120 @@
+import pg from 'pg';
+import config from '../config.js';
+
+module.exports=function(){
+
+const conString = `postgres://${config.userName}:${config.password}@${config.dbServerLocation}:${config.serverPort}/${config.dbName}`;
```

Comment:
Do you mean I should create a separate method for each task, or create a new script file for "initialization"?